vulcanai/models/utils.py

```diff
@@ -16,6 +16,7 @@
 def get_probs(network, loader, index_to_iter, ls_feat_vals):
```

Comment:
@joemehltretter `get_probs` should also be a private function which should be preceded by an underscore

Reply:
done

Reply:
thanks. sorry can you also do filtered_match_subj? somehow it slipped my mind

Reply:
I just did it @rfratila @joemehltretter